projects/assets/src/translations/en/user.ts

```diff
@@ -81,5 +81,9 @@ export const user = {
     confirmThatRead: 'I am confirming that I have read and agreed with the',
     termsAndConditions: 'Terms & Conditions',
     continue: 'Continue',
+    emailIsRequired: 'Invalid email format',
+    emailsMustMatch: 'Email addresses do not match',
+    termsAndConditionsIsRequired:
```

Comment:
Can you remove this please

Reply:
Done.

Reply:
Yes and I missed like 81 and 82 of the same file please